pkg/reconciler/volumeclaim/pvchandler.go

```diff
@@ -128,30 +128,29 @@ func (c *defaultPVCHandler) PurgeFinalizerAndDeletePVCForWorkspace(ctx context.C
 	return nil
 }
 
-func getPVCsWithoutAffinityAssistant(workspaceBindings []v1.WorkspaceBinding, ownerReference metav1.OwnerReference, namespace string) map[string]*corev1.PersistentVolumeClaim {
-	claims := make(map[string]*corev1.PersistentVolumeClaim)
-	for _, workspaceBinding := range workspaceBindings {
-		if workspaceBinding.VolumeClaimTemplate == nil {
-			continue
-		}
-
-		claim := workspaceBinding.VolumeClaimTemplate.DeepCopy()
-		claim.Name = GetPVCNameWithoutAffinityAssistant(workspaceBinding.VolumeClaimTemplate.Name, workspaceBinding, ownerReference)
-		claim.Namespace = namespace
-		claim.OwnerReferences = []metav1.OwnerReference{ownerReference}
-		claims[workspaceBinding.Name] = claim
+// getPVCFromVolumeClaimTemplate returns a PersistentVolumeClaim based on given workspaceBinding (using VolumeClaimTemplate), ownerReference and namespace
+func (c *defaultPVCHandler) getPVCFromVolumeClaimTemplate(workspaceBinding v1.WorkspaceBinding, ownerReference metav1.OwnerReference, namespace string) *corev1.PersistentVolumeClaim {
+	if workspaceBinding.VolumeClaimTemplate == nil {
+		c.logger.Infof("workspace binding %v does not contain VolumeClaimTemplate, skipping creating PVC", workspaceBinding.Name)
```

Comment:
nit: this log is misleading since this function doesn't create PVCs; can you rephrase or remove?

Reply:
Removed